_pages/decide/task-flow-analysis.md

```diff
@@ -6,27 +6,31 @@ redirect_from: /task-flow-analysis/
 
 ## What it is
 
-A step-by-step analysis of a common task a user must perform that diagrams the various touch points and decision points a user goes through to accomplish the task. The touch points should be represented as steps taken by the user, as well as steps taken by the system.
+A step-by-step analysis of how a user will interact with a system in order to reach a goal. This analysis is documented in a diagram that traces a user's possible paths through sequences of tasks and decision points in pursuit of their goal. The tasks and decision points should represent steps taken by the user, as well as steps taken by the system.
 
 ## Reasons to use it
 
-To illustrate in a solution-agnostic way the overall flow that a user progresses through to accomplish a single task. Task flow analysis also demonstrates the relationship between tasks, and how they interconnect across a site.
+To validate a design team's understanding of users' goals, common scenarios, and tasks, and to illustrate in a solution-agnostic way the overall flow of tasks through which a user progresses to accomplish a goal.  Task flow diagrams also help surface obstacles in the way of users achieving their goal. 
+
+## Prerequisites
+
+**User research.** Task flow analysis requires that you have a baseline understanding of your users' knowledge, goals, and behaviors. 
 
 ## Time required
 
-**Medium:** 1-2 hours per task
+**Medium:** 2-3 hours per user goal
 
 ## How to do it
 
-1. Identify the task(s) that need to be analyzed.
+1. Based on user research, identify target users' goals that need to be analyzed.
 
-2. Break each high-level task down into the subtasks and decisions that the user or system must perform. Specify the subtask in terms of objectives. Across all subtasks, you should cover the whole area of interest. Don’t make assumptions about which steps are understood.
+2. For each goal, identify common scenarios and the tasks and decisions that the user or system will perform in each scenario. Don’t make assumptions about which steps are understood.
```

Comment:
Good catch. I think the initial intent may have been to discourage the design team from just including tasks in the flow without breaking them down into component sub tasks, or asking users exactly what the tasks entail. We could either delete the sentence, or change it to something like this, but shorter: "Avoid assuming you and your stakeholders share a common understanding of the tasks. The idea with task flow diagrams is to make the flow of tasks explicit and then use the diagrams to check your understanding with users and subject matter experts."